pkg/storage/bucket/object_client_adapter.go

```diff
@@ -37,19 +66,14 @@ func NewObjectClientAdapter(bucket, hedgedBucket objstore.Bucket, logger log.Log
 		},
 	}
 
-	for _, opt := range opts {
```

Comment:
IIUC we are removing this because we removed the ClientOptions right?

Reply:
yes. It was being used to apply retry strategy from the provider specific adapters, but that bit is moved to the [init func](https://github.com/grafana/loki/pull/14867/files#diff-4053fe5568d51062c5213f4870cb80650f240cec3139d4aecb454bb3c72ba510R70)